apps/meteor/app/livechat/imports/server/rest/departments.ts

```diff
@@ -87,6 +87,22 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'livechat/department/isDepartmentCreationAvailable',
```

Comment:
So we have to be more worried about path collisions now :(

This is something that would be great to be shared somewhere so no one gets bitten by it.

Reply:
why we should be **more**  worried? to you have any extra explanation in mind? now at least we follow the same rules as express, it should be easy to find how it works

Reply:
Oh, because we apparently used to support things like 
path/:_id
path/something

Which express doesn't iirc.